node_modules/gh-auth/tests/util.js

```diff
@@ -24,6 +24,65 @@ var TestsUtil = require('gh-tests/lib/util');
 var UsersTestsUtil = require('gh-users/tests/util');
 
 /**
+ * Assert a user can log in
+ *
+ * @param  {RestClient}     client                  The REST client to make the request with
+ * @param  {String}         username                The username for the user
+ * @param  {String}         password                The password for the user
+ * @param  {Function}       callback                Standard callback function
+ */
+var assertLogin = module.exports.assertLogin = function(client, username, password, callback) {
+    client.auth.login(username, password, function(err, body, response) {
+        assert.ok(!err);
+
+        // Verify we're logged in
+        client.user.getMe(function(err, body, response) {
+            assert.ok(!err);
+            assert.strictEqual(body.anon, false);
+            return callback();
+        });
+    });
+};
+
+/**
+ * Assert a user can not log in
+ *
+ * @param  {RestClient}     client                  The REST client to make the request with
+ * @param  {String}         username                The username for the user
+ * @param  {String}         password                The password for the user
+ * @param  {Function}       callback                Standard callback function
```

Comment:
When I was writing this, I was having trouble with passport return the proper response code, I think I left it out for that reason